### DISCUSSION TITLE
[Esercitazione 13 Dicembre 2023] Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#323] [Esercizi - Viaggi/Piazza]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/767e2cba-b59c-45f5-baa4-f0739c5f2939)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/4e06ceae-5614-47c8-a048-97e30c075a10)

**Reply by Frisbon** [reply]
Confermo soluzione esatta. 
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/cb050549-8b5a-44e8-84c8-78bd366e24a2)
(moltiplicando per l'inversa dovrei ottenere la diagonale)
